.travis.yml

```diff
@@ -31,11 +31,23 @@ addons:
     - conntrack
 script: |
   "${MAKE}" unit build
-  if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-    sudo minikube start # vm-driver=none requires root
-    sudo chown -R "$USER" "$HOME/.kube" "$HOME/.minikube"
-    KUBECONFIG="$HOME/.kube/config" make e2e
+  if [[ "$TRAVIS_OS_NAME" != "linux" ]]; then
+    echo "skipping e2e tests for ${TRAVIS_OS_NAME}"
```

Comment:
so we're publishing binaries for all 3 platforms but only running e2e tests against linux? The other platforms are supported but not tested against? Should we make a note of that in the docs.